examples/v2-minimal/packages/contracts/types/ethers-contracts/IWorld.ts

```diff
@@ -472,11 +501,13 @@ export interface IWorldInterface extends utils.Interface {
 
   events: {
     "StoreDeleteRecord(bytes32,bytes32[])": EventFragment;
+    "StoreEphemeralSetRecord(bytes32,bytes32[],bytes)": EventFragment;
```

Comment:
we should probably align the names of these
- `StoreEphemeralSetRecord` event
- `setEphemeralRecord` function

Reply:
in terms of API, I am wondering if it would be more clear to use the term "emit" instead of "set" and just call it an ephemeral record, e.g.
- `StoreEphemeralRecord` (since there is no notion of set/delete/field updates, etc)
- `emitEphemeralRecord` (rather than set)

Reply:
i agree with emit